viewer/packages/data-providers/src/pointcloud-stylable-object-providers/LocalPointCloudStylableObjectProvider.ts

```diff
@@ -4,11 +4,11 @@
 
 import { ModelIdentifier } from '@reveal/data-providers';
 import { PointCloudStylableObjectProvider } from '../PointCloudStylableObjectProvider';
-import { PointCloudObjectData } from './PointCloudObjectAnnotationData';
+import { PointCloudObject } from './types';
 
 export class LocalPointCloudStylableObjectProvider implements PointCloudStylableObjectProvider {
-  async getPointCloudObjects(_modelIdentifier: ModelIdentifier): Promise<PointCloudObjectData> {
+  async getPointCloudObjects(_modelIdentifier: ModelIdentifier): Promise<PointCloudObject[]> {
     // Currently, we don't support annotations in a local models
-    return new PointCloudObjectData([]);
+    return [];
   }
 }
```

Comment:
Why does this class exist?

Reply:
It's used in tests and for the local point cloud manager. I renamed it to `DummyPointCloudStylabelObjectProvider` now to make it more clear what it does (which is nothing).